layers/+completion/auto-completion/funcs.el

```diff
@@ -294,8 +294,14 @@ MODE parameter must match the :modes values used in the call to
       (define-key map (kbd "C-j") 'company-select-next)
       (define-key map (kbd "C-k") 'company-select-previous)
       (define-key map (kbd "C-l") 'company-complete-selection))
-    (when (require 'company-quickhelp nil 'noerror)
-      (evil-define-key 'insert company-quickhelp-mode-map (kbd "C-k") 'company-select-previous)))
+    ;; Fix company-quickhelp Evil C-k
+    (defun spacemacs//set-C-k-company-select-previous (&rest args)
+      (define-key evil-insert-state-map (kbd "C-k") 'company-select-previous))
+    (defun spacemacs//restore-C-k-evil-insert-digraph (&rest args)
+      (define-key evil-insert-state-map (kbd "C-k") 'evil-insert-digraph))
```

Comment:
that's really a stupid change...

Reply:
@yzprofile Do you have a suggestion for a better solution?

This is the issue it's trying to fix:
Completion: C-k not working consistently #2974

Reply:
I mean do not override `C-k` in `company-completion-finished-hook` and `company-completion-cancelled-hook ` or make it as optional.

Reply:
Ok, that's more descriptive. A tip for future comments, add more details in the first comment:
>This is stupid...
Because...

😄 

#### This is what I'm seeing with:
`(auto-completion :variables auto-completion-enable-help-tooltip t)`

#### Without this change.
- On an empty line in `.spacemacs`
- Type `dot`

Now a drop down list (company-mode) opens with suggestions.

Before a tooltip has appeared
- `C-j` moves down in the list
- `C-k` moves up in the list

When the tooltip has appeared, then `C-k` inserts `?` (it called `evil-insert-digraph`).
It waits for two key presses, for example: if one types `12` then it inserts the character: ½

#### With this change
`C-k` continues to navigate up in the list after the tooltip has been shown.
To insert a digraph character with `C-k`, the drop down list has to be closed first with `C-g`.

--- 

Could you explain what your observing and expecting.
And include your system info (`SPC h d s` copies it to your clipboard).